nav2_map_server/src/map_saver/main_cli.cpp

```diff
@@ -1,3 +1,4 @@
+// Copyright 2020 Shivam Pandey pandeyshvivamm2017robotics@gmail.com
```

Comment:
Are you sure that new copyrights are necessary here? There is no new principal logic or algorithm were added into this file, and it was not also completely re-worked comparing to initial state (I do hope).

Reply:
No need for that, will remove it.